docs/manual/constant-value-checker.tex

```diff
@@ -251,6 +251,39 @@ \section{Warnings\label{value-checker-warnings}}
 
 \end{itemize}
 \end{sloppypar}
+
+
+\section{Overflow\label{value-checker-overflow}}
+
+The Constant Value Checker normally takes Java's normal overflow rules into account when
+computing the possible values of expressions.
+The \code{-AignoreRangeOverflow} command-line option
+makes it ignore the possibility of overflow
+for range annotations (\<@IntRange>, for instance), making them more precise.
+You can supply the \code{-AignoreRangeOverflow} command-line option
+when you are confident that no overflow will occur in the program (for
```

Comment:
I agree that this makes the Value Checker unsound - that's the whole point. The Index Checker is built on the assumption that it doesn't have to check overflow, and integrating the significant parts of the Index Checker into the Value Checker means that we need that assumption to be an option in the Value Checker. That's the fundamental reason for this change. I'm not sure how else to describe that in the manual, though - maybe the flag just shouldn't be user visible? I can make that change.